xpdview/viewer_mpl.py

```diff
@@ -0,0 +1,25 @@
+"""module to start pure mpl-backend viewer"""
+import matplotlib.pyplot as plt
+import numpy as np
+from .cross_2d import StackViewer, CrossSection
+from .waterfall import Waterfall
+
+# init
+viewer_fig = plt.figure()  # let mpl decide backend
+cross_section = CrossSection(viewer_fig)
+stack_viewer = StackViewer(cross_section)
+
+waterfall_fig = plt.figure()  # let mpl decide backend
+waterfall = Waterfall(waterfall_fig, waterfall_fig.canvas)
+
+# def data list to test
```

Comment:
Is this data going into the classes defined above?

Reply:
Correct, data should be passed to init.

This file was used for my local test and in fact it's included to git by accident. But since I already did that, I will turn it into as proper test file for both waterfall and stack viewer